lua/core/mapping.lua

```diff
@@ -24,30 +26,45 @@ local core_map = {
 	["n|<A-]>"] = map_cr("vertical resize +5"):with_silent():with_desc("window: Resize +5 vertically"),
 	["n|<A-;>"] = map_cr("resize -2"):with_silent():with_desc("window: Resize -2 horizontally"),
 	["n|<A-'>"] = map_cr("resize +2"):with_silent():with_desc("window: Resize +2 horizontally"),
-	["n|<C-q>"] = map_cmd(":wq<CR>"):with_desc("editn: Save file and quit"),
-	["n|<A-S-q>"] = map_cmd(":q!<CR>"):with_desc("editn: Force quit"),
-	["n|<leader>o"] = map_cr("setlocal spell! spelllang=en_us"):with_desc("editn: Toggle spell check"),
+	["n|<C-q>"] = map_cmd(":wq<CR>"):with_desc("edit: Save file and quit"),
+	["n|<A-S-q>"] = map_cmd(":q!<CR>"):with_desc("edit: Force quit"),
+	["n|<leader>o"] = map_cr("setlocal spell! spelllang=en_us"):with_desc("edit: Toggle spell check"),
+	["n|+"] = map_cmd("<C-a>"):with_noremap():with_silent():with_desc("edit: Increment"),
+	["n|-"] = map_cmd("<C-x>"):with_noremap():with_silent():with_desc("edit: Decrement"),
+	["n|<C-a>"] = map_cmd("gg<S-v>G"):with_noremap():with_silent():with_desc("edit: Select all"),
```

Comment:
```suggestion
	["n|<C-a>"] = map_cmd("gg0vG$"):with_noremap():with_silent():with_desc("edit: Select all"),
```

Perhaps using visual mode would be better

Reply:
u mean "v|<C-a>"?

Reply:
Just literally _visual_ mode (by contrast with visual **block** mode) 😃

Reply:
oh got it lol